frontend/schemaCMS/src/routes/dataSource/metadata/__tests__/__snapshots__/metadata.component.spec.js.snap

```diff
@@ -753,6 +753,7 @@ exports[`Metadata: Component should render correctly 1`] = `
     className="sc-gtssRu"
   >
     <div
+      id=""
```

Comment:
We shouldn't have empty ids in html elements. It won't make any harm here, but it's a good practice not to add them.